### ISSUE TITLE
Explorer view

### ISSUE THREAD

**Issue by kaixoo** [friction2d/friction#269]
In the mid term, an explorer view could be added like in Adobe for storing commonly used animations. I'm not an After Effects user but I think this dialog would be useful anyway. Of course this could be hidden from the menu.

**Comment by rodlie** [comment]
Please be more specific, "like Adobe" means nothing to me 😄 (I don't use Adobe software).

**Comment by kaixoo** [comment]
Sorry. I don't neither.

I can't find the exact dialog right now. I meant this kind of file view where you can share animations or scripts between multiple projects.

![imatge](https://github.com/user-attachments/assets/8ee997c7-f1e5-47b9-a5b3-c92370e681d6)

**Comment by kaixoo** [comment]
I think this is a synonymous to Friction's Assets dialog but with improvements. 

![image](https://github.com/user-attachments/assets/5ca7644b-c81a-4f60-88b8-2b2d67c59ddf)

P.S: Why are there 2 places to change properties? It's redundant and confusing.

![image](https://github.com/user-attachments/assets/6e1aba59-9815-442a-a9f6-12c8baa1379b)

**Comment by rodlie** [comment]
> I think this is a synonymous to Friction's Assets dialog but with improvements.

I'm aware of that. "assets" is not a high priority for me, never use it. If it was up to me this would be a popup action in the toolbar.

> P.S: Why are there 2 places to change properties? It's redundant and confusing.

You will figure that out when you get large/complicated project.

**Comment by kaixoo** [comment]
I agree ofc. First the structural issues. But afterwards in the project a folder tree view could be handy for organization.

**Comment by rodlie** [comment]
We can always expand the assets widget (with a custom delegate or something) with more info/etc, I have no issues with that. Currently it's a MVP.

Please list wanted options/features for discussion.

**Comment by kaixoo** [comment]
It seems this feature should be just a file explorer you can drag from / to files. Definitely not a priority but it's handy when handling a project with multiple scenes and don't want to bury everything to the same animation file. Also can be useful for saving presets for scripts, assets and prefabs.
